[Pages] deploy hooks revision (#2005)
cloudflare/cloudflare-docs: products/pages/src/content/platform/deploy-hooks.md

- Implement a fully customized CI/CD pipeline, deploying only under desired conditions 
- Schedule a CRON trigger to update your website on a fixed timeline

> To create a Deploy Hook, navigate from **Account Home** to **Pages** and select your desired Pages project. Navigate to **Settings** > **Builds & deployments** and select **Add deploy hook** to start configuration.

Reviewer: Should this be a numbered list of steps?

![deploy hooks](deploy-hooks-add.png)

## Parameters needed
To configure your Deploy Hook, you must enter two key parameters:
   1. **Deploy hook name:** a unique identifier for your Deploy Hook (e.g. “contentful-site”)
   2. **Branch to build:** the repository branch your Deploy Hook should build

![deploy hooks](deploy-hooks-configure.png)

## Using your Deploy Hook
Once your configuration is complete, the Deploy Hook’s unique URL is ready to be used. You will see both the URL as well as the POST request snippet available to copy. 
![deploy hooks](deploy-hooks-details.png)

> Every time a request is sent to your Deploy Hook, a new build will be triggered. Review your deployment log to note that any deployment triggered by a Deploy Hook will specify this in the source column.  

Reviewer: ```suggestion
Every time a request is sent to your Deploy Hook, a new build will be triggered. Review the **Source** column of your deployment log to see which deployment were triggered by a Deploy Hook.  
```
